document/core/util/macros.def

```diff
@@ -873,7 +873,7 @@
 
 .. Stack, meta functions
 
-.. |expand| mathdef:: \xref{exec/runtime}{syntax-frame}{\F{expand}}
+.. |expand| mathdef:: \xref{exec/runtime}{exec-expand}{\F{expand}}
```

Comment:
Ha, yeah, I didn't notice I still had that in my tree. One could argue it's somewhat related to typeuse, though, since it expands type uses in block types. :)